model/src/main/proto/exploration.proto

```diff
@@ -322,6 +322,39 @@ message UserAnswer {
   WrittenTranslationContext written_translation_context = 6;
 }
 
+// Corresponds to a raw item selection answer that user has submitted.
+message ItemSelectionRawAnswer {
+  repeated int32 selected_indexes = 1;
+}
+
+// Corresponds to a raw user answer that user has submitted.
```

Comment:
This needs a bit more detail to explain what is meant by 'raw user answer' and how this should be used.

Reply:
Updated comment, thanks.